cmd/hzpaste-api/main.go

```diff
@@ -71,9 +64,25 @@ func getFileName() string {
 	if len(env) == 0 {
 		env = "development"
 	}
-	filename := []string{"internal/config/", "config.", env, ".json"}
-	_, dirname, _, _ := runtime.Caller(0)
-	filePath := path.Join(filepath.Dir(dirname), strings.Join(filename, ""))
+	filename := []string{"config/", "config.", env, ".json"}
+	filePath := path.Join(strings.Join(filename, ""))
 
 	return filePath
 }
+
+func initConfig() Configuration {
+	configuration := Configuration{}
+	host := os.Getenv("HZPASTE_HOST")
```

Comment:
Since you're setting values from the env outside the condition block, and configuration fields already are being parsed (see struct definition), we could get the configuration partially initialized from env values and partially from configuration file.